community-operators/strimzi-kafka/strimzi-cluster-operator.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,545 @@
+          - delete
+          - patch
+          - update
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          verbs:
+          - get
+          - create
+          - delete
+          - patch
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - get
+        - apiGroups:
+          - kafka.strimzi.io
+          resources:
+          - kafkatopics
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - patch
+          - update
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+        - apiGroups:
+          - kafka.strimzi.io
+          resources:
+          - kafkausers
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - patch
+          - update
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - get
+          - list
+          - create
+          - patch
+          - update
+          - delete
+        - apiGroups:
+          - kafka.strimzi.io
+          resources:
+          - kafkatopics
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - patch
+          - update
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+        - apiGroups:
+          - "kafka.strimzi.io"
+          resources:
+          - kafkas
+          - kafkaconnects
+          - kafkaconnects2is
+          - kafkamirrormakers
+          - kafkausers
+          - kafkatopics
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+          - patch
+          - update
+      deployments:
+      - name: strimzi-cluster-operator
+        spec:
+          replicas: 1
+          template:
+            metadata:
+              labels:
+                name: strimzi-cluster-operator
+            spec:
+              serviceAccountName: strimzi-cluster-operator
+              containers:
+              - name: strimzi-cluster-operator
+                image: strimzi/cluster-operator:latest
+                imagePullPolicy: IfNotPresent
+                env:
+                - name: STRIMZI_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
+                  value: "120000"
+                - name: STRIMZI_OPERATION_TIMEOUT_MS
+                  value: "300000"
+                - name: STRIMZI_DEFAULT_ZOOKEEPER_IMAGE
+                  value: strimzi/zookeeper:latest-kafka-2.0.0
+                - name: STRIMZI_KAFKA_IMAGES
```

Comment:
This seems to be taken from master. The actual deployment for 0.9.0 is here. You should replace the whole `env` section with this:

```yaml
        env:
        - name: STRIMZI_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
          value: "120000"
        - name: STRIMZI_OPERATION_TIMEOUT_MS
          value: "300000"
        - name: STRIMZI_DEFAULT_ZOOKEEPER_IMAGE
          value: strimzi/zookeeper:0.9.0
        - name: STRIMZI_DEFAULT_KAFKA_IMAGE
          value: strimzi/kafka:0.9.0
        - name: STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE
          value: strimzi/kafka-connect:0.9.0
        - name: STRIMZI_DEFAULT_KAFKA_CONNECT_S2I_IMAGE
          value: strimzi/kafka-connect-s2i:0.9.0
        - name: STRIMZI_DEFAULT_KAFKA_MIRRORMAKER_IMAGE
          value: strimzi/kafka-mirror-maker:0.9.0
        - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
          value: strimzi/topic-operator:0.9.0
        - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
          value: strimzi/user-operator:0.9.0
        - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
          value: strimzi/kafka-init:0.9.0
        - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
          value: strimzi/zookeeper-stunnel:0.9.0
        - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
          value: strimzi/kafka-stunnel:0.9.0
        - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
          value: strimzi/entity-operator-stunnel:0.9.0
        - name: STRIMZI_LOG_LEVEL
          value: INFO
```